gramps/plugins/db/dbapi/dbapi.py

```diff
@@ -480,6 +480,34 @@ def get_person_handles(self, sort_handles=False, locale=glocale):
             self.dbapi.execute("SELECT handle FROM person")
         return [row[0] for row in self.dbapi.fetchall()]
 
+    def get_person_cursor(self, sort_handles=False, locale=glocale):
```

Comment:
```suggestion
    def get_person_handles_cursor(self, sort_handles=False, locale=glocale):
```
For consistency with `get_person_handles`, should this method be called `get_person_handles_cursor`?